src/styles/calendar.scss

```diff
@@ -0,0 +1,50 @@
+
+.calendar {
+  width: $calendar--size;
+  display: flex;
+  flex-wrap: wrap;
+  gap: $day-block--gap;
+  padding: 10px;
+
+  @each $day, $margin in $days {
+    &--start-day--#{$day} {
+      .calendar__day:first-child {
```

Comment:
```suggestion
      &__day:first-child {
```

Reply:
I have tried to use `&__day` but it is not working, and I don't understand why.